.prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
+   "trailingComma": "es5",
+   "tabWidth": 2,
+   "semi": true,
+   "singleQuote": true
+ }
+
+ 
```

Comment:
Dont need the extra blank lines at the end here. That said, Since prettier isnt a dependency for the project should this file be here? By comparison eslint is a dependency, so it makes sense to have its config included